apify-docs-theme/src/theme/Navbar/Content/index.jsx

```diff
@@ -35,6 +36,14 @@ function NavbarContentLayout({
     );
 }
 
+function SubNavbarTitle({ titleIcon, title }) {
```

Comment:
icon next to the title of the subnavbar. This will be released in docs-theme package and then we need to add it actually to js-sdk, python-sdk, js-api-client and python-api-client
As a consequence we have a "duplicate" of javascript and python 40x40 svgs in the docs-theme static/img folder